### ISSUE TITLE
This is fairly complex, you can probably simplify it. One way would be to always have `obj` be a list so you don't need the `isinstance` check.

### ISSUE THREAD

**Issue by jengroff** [jengroff/participant-screener#4]
This is fairly complex, you can probably simplify it. One way would be to always have `obj` be a list so you don't need the `isinstance` check.

Give me some example responses and expected outputs and happy to give it a crack. 

Also do we need nested functions here?

_Originally posted by @bbelderbos in https://github.com/jengroff/participant-screener/pull/2#r677594923_

**Comment by bbelderbos** [comment]
Yes this you ideally would refactor, let me know if you need help.
